ts-tests/tests/test-balance.ts

```diff
@@ -12,6 +12,7 @@ describeWithFrontier("Frontier RPC (Balance)", (context) => {
 	});
 
 	step("balance to be updated after transfer", async function () {
+		await createAndFinalizeBlock(context.web3);
```

Comment:
This should go after the timeout line.
Also why do we need to create a block for this ?